vendor/riscv/riscv-isa-sim/Makefile.in

```diff
@@ -96,7 +96,7 @@ VPATH := $(addprefix $(src_dir)/, $(sprojs_enabled))
 # highest.
 SPIKE_HASH_VERSION:= $(shell git log -1 --pretty=tformat:"%h" -- .. )
```

Comment:
This pertains to the previous commit, and resolving it would be advantageous as well.

The command `git log -1 --pretty=tformat:"%h"` may produce either 7 or 8 characters depending on the specific Git version being used.

For instance, Git 1.8.3.1 outputs 7 characters, whereas Git 2.44.0 outputs 8 characters.

Therefore, relying solely on this command might lead to errors if there are variations in the Git version used, especially when different versions are loaded via module load.

To ensure consistency and accuracy across all Git versions, it's recommended to specify 8 characters for the abbreviated commit hash:

```
git log -1 --pretty=tformat:"%h" --abbrev=8
```

This will consistently provide an 8-character abbreviated commit hash, eliminating potential discrepancies between different Git versions.

Reply:
Makes sense, I will push the changes :+1:  Thanks!